packages/itwin/quantity-formatting/package.json

```diff
@@ -0,0 +1,87 @@
+{
+  "name": "@itwin/quantity-formatting-react",
+  "version": "0.0.0",
+  "license": "MIT",
+  "description": "React components and utilities for quantity formatting",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/iTwin/viewer-components-react/tree/master/packages/itwin/quantity-formatting"
+  },
+  "type": "module",
+  "types": "./lib/quantity-formatting-react.d.ts",
+  "exports": {
+    ".": {
+      "types": "./lib/quantity-formatting-react.d.ts",
+      "default": "./lib/quantity-formatting-react.js"
+    }
+  },
+  "scripts": {
```

Comment:
might be nice to setup api-extractor in this pkg. not needed in this pr, but def needed before 1.0

Reply:
will you be creating an issue for this if you dont do it in this pr?